WooCommerce/Classes/Authentication/Store Creation/StoreCreationCoordinator.swift

```diff
@@ -744,7 +744,7 @@ private extension StoreCreationCoordinator {
                 self.storeCreationProgressViewModel?.incrementProgress()
             })
             // Retries 10 times with some seconds pause in between to wait for the newly created site to be available as a Jetpack/Woo site.
```

Comment:
Super nit: Maybe the comment needs to be updated too?

Reply:
Good point! updated in https://github.com/woocommerce/woocommerce-ios/pull/9957/commits/ee249165551a2865b5e9f4ed55172bca8571a8d8